app/views/notifier/new_user.text.haml

```diff
@@ -1,7 +1,17 @@
-= text(".intro", first_name: @user.first_name, last_name: @user.last_name, login_url: new_user_session_url)
+= text(".intro_text", first_name: @user.first_name, last_name: @user.last_name, login_url: new_user_session_url)
+= "" # new line
```

Comment:
There might be, but I couldn't figure out anything less stupid that worked haha.